lib/dependencies/ImportParserPlugin.js

```diff
@@ -46,6 +46,8 @@ class ImportParserPlugin {
 			Array.from(enumerable, e => [e]);
 		parser.hooks.importCall.tap("ImportParserPlugin", expr => {
 			const param = parser.evaluateExpression(expr.source);
+			// TODO support deferImport
+			// this requires the evaluateExpression to support Object literals.
```

Comment:
@Jack-Works So we don't support `import('something', { defer: true })`? Do we have this in spec?

Reply:
> @Jack-Works So we don't support `import('something', { defer: true })`? Do we have this in spec?

No, the syntax becomes `import.defer('something')` (this comment is written in a very early stage), I'll update this soon.

Reply:
Thank you